core/code/portal_marker.js

```diff
@@ -10,14 +10,15 @@ var portalBaseStyle = {
 
 // portal hooks
 function handler_portal_click (e) {
-  e.target.select(true);
+  window.selectPortal(e.target.options.guid, e.type);
+  window.renderPortalDetails(e.target.options.guid, true)
 }
 function handler_portal_dblclick (e) {
-  e.target.select(true);
+  handler_portal_click(e);
```

Comment:
BAD!

Reply:
what is so bad for you to capslock ?

Reply:
click/dblclicl/contexmenu - are separate actions.
Don't call one from another, even if currently it does the job.
Better make separate function for these calls:
```
  window.selectPortal(e.target.options.guid, e.type);
  window.renderPortalDetails(e.target.options.guid, true)
```  
Also I suspect that we we can refactor code to leave only one of these.